docker-compose.yml

```diff
@@ -18,3 +18,6 @@ db:
     - "5432"
   environment:
       POSTGRES_DB: champaign
+
+redis:
+  image: redis
```

Comment:
Sorry, this might be a pinch confusing (and could possibly be fixed in a cleanup), but CircleCI uses a docker-compose.yml template in its own directory (because there's a tag that needs to be replaced with the git commit SHA with envsubst). You can find it in `circle/docker-compose-template.yml`. If you need to debug this on CircleCI, you can SSH into a running build - there's a select to rebuild on the page where the build is, and you can select 'rebuild with SSH' from the dropdown. That'll give you the details you need.